apps/frontend/pages/dashboard/saved.tsx

```diff
@@ -0,0 +1,24 @@
+import { Layout } from "../../components/Layout/Layout";
+import { ActionButton } from "../../components/ActionButton/ActionButton";
+import { Results } from "../../views/Results/Results";
+import { AuthChecker } from "../../components/AuthChecker/AuthChecker";
+import { useEffect } from "react";
+import { UserAPI } from "../../lib/api/user";
+import { useDispatch } from "react-redux";
+
+const Saved = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(UserAPI.getSavedOffers());
```

Comment:
Czemu robisz ten dispatch tutaj, skoro ten komponent nie potrzebuje żadnych danych?

Reply:
Bo chciałbym, żeby od razu po załadowaniu tej strony było to wykonywane, a jak inaczej powinienem to zrobić?